### DISCUSSION TITLE
brew formula for tty0tty

### DISCUSSION THREAD

**Discussion by xeonfusion** [Homebrew/discussions#2852] [Everyday usage]
Is it possible to create a brew formula for tty0tty https://github.com/freemed/tty0tty that creates kernel modules to emulate virtual serial ports in Mac OS X?

**Comment by SMillerDev** [comment]
Probably, https://docs.brew.sh/Adding-Software-to-Homebrew documents how

**Reply by carlocab** [reply]
It looks like it depends on the Linux kernel specifically, unfortunately: https://github.com/freemed/tty0tty/blob/8f5842c18ebe61f18d0213e0b5c133c174c68523/module/tty0tty.c#L28-L41

We could probably package it for Linux, though.
